electron-app/src/store/balances/getters.ts

```diff
@@ -112,6 +114,33 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, Zero);
   },
 
+  manualBalances: (state: BalanceState) => {
+    return state.manualBalances;
+  },
+
+  manualBalanceByLocation: (state: BalanceState) => {
+    const simplifyManualBalances = state.manualBalances.map(obj => {
+      const newObj: ManualBalancesByLocation = { location: '', usdValue: Zero };
```

Comment:
Since the initial values endup being overwritten, maybe you can do a destructure `const { localtion, usdValue } = obj` and then `return { location, usdValue }` instead. Also maybe instead of obj it would be better to have a more descripting name?

Reply:
Good points, fixed!